roles/contiv_network/tasks/ovs.yml

```diff
@@ -52,3 +56,11 @@
   with_items:
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
+
+- name: disable SE linux
```

Comment:
I think for development images it's fine, but maybe we should leave it on just to ensure we support operation under it, people are going to want it.